OpenSim/Common/STOFileAdapter.h

```diff
@@ -174,6 +174,9 @@ createSTOFileAdapterForReading(const std::string& fileName);
 std::shared_ptr<DataAdapter>
 createSTOFileAdapterForWriting(const AbstractDataTable& table);
 
+typedef STOFileAdapter_<double> STOFileAdapter;
+typedef STOFileAdapter_<SimTK::Vec3> STOFileAdapterVec3;
```

Comment:
Is this change related to the rest of the PR? I don't see these classes being used in the luxo example.